packages/fiori/src/Timeline.js

```diff
@@ -16,6 +18,28 @@ const metadata = {
 	tag: "ui5-timeline",
 	languageAware: true,
 	managedSlots: true,
+	properties: /** @lends  sap.ui.webcomponents.fiori.Timeline.prototype */ {
+		/**
+		 * Select orientationn.
```

Comment:
typo -> orientationn

and could be more descriptive: "Defines the items orientation"

Reply:
Done